components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/PublisherCommonUtils.java

```diff
@@ -2411,8 +2420,21 @@ public static API addAPIWithGeneratedSwaggerDefinition(API apiToAdd, String oasV
             }
             apiToAdd.setSwaggerDefinition(apiDefinition);
         } else {
-            AsyncApiParser asyncApiParser = new AsyncApiParser();
+            /**
+             * Note: By default, AsyncAPI version 3.0 will be used when creating new streaming APIs
```

Comment:
we dont need this comment here. This is something which should go to the AsyncApiParserFactory class level or method level description